nansat/vrt.py

```diff
@@ -696,19 +696,21 @@ def _update_warped_vrt_xml(self, x_size, y_size, geo_transform, block_size, work
 
     def _create_band_name(self, dst):
         """Create band name based on destination band dictionary <dst>"""
-        wkv = {}
         band_name = dst.get('name', None)
 
+        # try to get metadata from WKV using PyThesInt if it exists
+        wkv_dst = dst.get('wkv', None)
+        try:
+            wkv_pti = pti.get_wkv_variable(str(wkv_dst))
+        except IndexError:
+            wkv = {}
+        else:
```

Comment:
I'm not sure about that. Do you expect any other exceptions? If so then maybe it worth to specify them or just move `wkv = dict(wkv_pti)` to the `try`. Could you explain it, please?